lib/helpers.js

```diff
@@ -338,4 +340,11 @@ exports.truncateDecimals = function (number, digits) {
   return truncatedNum / multiplier;
 };
 
+exports.escapeSpace = function (str) {
+  if(support.util.hasValue(str)) {
+    return str.split(SPACE).join(ESCAPE_SPACE_RE);
+  } else {
+    throw new Error("Could not find the xcodePath");
```

Comment:
this error message doesn't make sense in this method; this method has nothing to do with xcodePath anymore. i think we don't even need to do a value check here at all. if we don't have a value in `str`, calling `.split()` on it will throw an error anyway. so this whole method can be one line.